syscallutil/syscallutil_test.go

```diff
@@ -0,0 +1,45 @@
+package syscallutil
+
+import (
+	"fmt"
+	"runtime"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestLoadLibrary(t *testing.T) {
+	t.Run("Test valid library", func(t *testing.T) {
+		var lib string
+		switch runtime.GOOS {
```

Comment:
We can use the helpers from `https://github.com/projectdiscovery/utils/blob/main/os/os.go`
```
osutil.IsWindows()
osutil.IsLinux()
...
```